### ISSUE TITLE
[A11Y][Android] Support high contrast mode on Chip component

### ISSUE THREAD

**Issue by pya35** [Orange-OpenSource/ouds-flutter#636]
### Prerequisites

- [x] I have [searched the backlog](https://github.com/Orange-OpenSource/ouds-flutter/issues) for duplicate or closed feature requests

### Your test device

S25 and iphone

### OS Version

android and ios

### App version

1.1.2

### Describe the issue

As already done for checkbox, switch and radio, chips should support high contrast mode

### Expected Behavior

Support high contrast mode on both apps

**Comment by nouha06** [comment]
@pya35  this should be applied for both suggestion chip and filter chip ? we can say that this [ticket](https://github.com/Orange-OpenSource/ouds-flutter/issues/494) is duplicated ?

**Comment by pya35** [comment]
@nouha06 you're right, i'm closing the ticket
